front/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM    php:7.3-apache
```

Comment:
keep nginx please

Reply:
okay, do you want nginx for all default web challenges in pathwar repo as well ?

Reply:
nop, just for this one

other levels require php, this one only hosts static files

Reply:
Understood !